src/deavinaSJun20/StringMatchingClass.java

```diff
@@ -0,0 +1,32 @@
+package deavinaSJun20;
+
+/* Print minimum rotation required to compare both strings equality.
+
+
+Example 1 :
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+*/
+public class StringMatchingClass {
+
+	int matchString(String input1, String input2) {
+		int rotation_count = 0;
+		while(! input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && rotation_count< input1.length)

Reply:
Thanks for suggestion. I will practice more and more.